code/modules/awaymissions/maploader/writer.dm

```diff
@@ -172,20 +172,10 @@
 	if(!use_json)
 		for(var/V in A.vars)
 			CHECK_TICK
-			if((!issaved(A.vars[V])) || (A.vars[V] == initial(A.vars[V])))
```

Comment:
Technically this was always true and always resulted in `continue`, so this has different behavior now.